scripts/setup.py

```diff
@@ -60,8 +60,13 @@ def main():
     # the script is creating the proof root
     proof_root = os.path.abspath('proofs')
 
-    # the script is linking to the litani script within the litani submodule
-    litani = util.read_litani_path()
+    # the script is linking to litani
+    if util.is_litani_installed():
+        litani = '"litani"'
+    else:
+        litani_path = util.read_litani_path()
+        relpath = os.path.relpath(litani_path, proof_root)
+        litani = f"$(abspath $(PROOF_ROOT)/{relpath})"
```

Comment:
You're still passing `litani` as a parameter to the function which I'm okay with because that is how it was before. I personally think this logic could just be handled within the function instead of being passed in as a parameter, so I'm curious as to if there's a reason its being done this way, but I'm not going to block on it

Reply:
I just abstracted this logic inside its own function as pointed out by Nick